geoengine/ml.py

```diff
@@ -6,10 +6,14 @@
 import tempfile
 from typing import Protocol
 from dataclasses import dataclass
-from geoengine_openapi_client.models import MlModelMetadata, MlModel
+from onnx.reference import ReferenceEvaluator
```

Comment:
unrelated to this line: why did you add "zipmap: false" to the ml_pipeline.ipynb? it worked without it

Reply:
(a) It seems to be not a favorable option: https://onnx.ai/sklearn-onnx/versions/v1.16.0/auto_examples/plot_convert_zipmap.html#let-s-compare-prediction-time
(b) it does not work with the ReferenceEvaluator

Reply:
[0df8a27](https://github.com/geo-engine/geoengine-python/pull/196/commits/0df8a27321e7cbebf477775d1741b20f61adc03b)